website/blog/2020-02-06-react-navigation-5.0.md

```diff
@@ -0,0 +1,172 @@
+- Simpler API for [`reset` action](https://reactnavigation.org/docs/navigation-prop.html#reset) where you can pass the new state directly instead of a chain of actions.
+- More reliable [`focus` and `blur` events](https://reactnavigation.org/docs/navigation-lifecycle.html) to know when a screen's focus state changes.
+- Integration with [`InteractionManager`](https://facebook.github.io/react-native/docs/interactionmanager) to delay tasks until animation is complete.
+- Better safe area handling with [`react-native-safe-area-context`](https://github.com/th3rdwave/react-native-safe-area-context).
+
+## Upgrading
+
+This is a big release and, while the basic concepts such as nesting are the same, the new API is mostly incompatible with the previous API. We know it can be a challenge to upgrade your code base. So we're going to keep supporting React Navigation 4 with bug fixes. We'll accept contributions and keep it compatible with the latest React Native version. The old code will now leave at [`react-navigation/react-navigation-4`](https://github.com/react-navigation/react-navigation-4) on GitHub.
+
+We recommend starting your new projects with the new version so you can take advantage of the new APIs and the new features.
+
+We have written an [upgrade guide](https://reactnavigation.org/docs/upgrading-from-4.x.html) which will give you an overview of what's changed and how to adapt the old API and concepts to the new API.
+
+## A note for alpha users
+
+If you were using React Navigation 5 when it was alpha, you might need to check the following changes when upgrading:
+
+- If you have added `@react-navigation/core` to your dependencies, remove it, and replace all imports from `@react-navigation/core` with `@react-navigation/native`
+- If you were importing `NavigationNativeContainer`, change it to `NavigationContainer`, if you were using `NavigationContainer`, change it to `BaseNavigationContainer`
+- If you had deep linking configured, the config format has changed for nesting. Check the [deep linking docs](https://reactnavigation.org/docs/deep-linking.html) for details.
+
+## Thanks to these wonderful people
+
+React Navigation 5 wouldn't have possible without these wonderful people. A lot of thanks to [Michał Osadnik](https://github.com/osdnk) for working in this project with me from idea to fruition, [Brent Vatne](https://github.com/brentvatne) for his ideas, encouragement and funding from [Expo](https://expo.io), [Wojciech Lewicki](https://github.com/WoLewicki) and [Jakub Gonet](https://github.com/jakub-gonet) for improving the documentation, and working on deep link support, [Krzysztof Magiera](https://github.com/kmagiera) and [Software Mansion](https://github.com/software-mansion) for their awesome libraries that the core functionality depends on and contibutions, [Janic Duplessis](https://github.com/janicduplessis) for [`react-native-safe-area-context`](https://github.com/th3rdwave/react-native-safe-area-context).
```

Comment:
_been_ is missing:

_React Navigation 5 wouldn't have **been** possible without these wonderful people._